components/compliance-service/reporting/relaxting/reports.go

```diff
@@ -513,6 +513,7 @@ func (backend *ES2Backend) GetReport(reportId string,
 					NodeName:         esInSpecReport.NodeName,
 					Environment:      esInSpecReport.Environment,
 					Status:           esInSpecReport.Status,
+					JobId:            esInSpecReport.JobID,
```

Comment:
i found we had the job id on the protos for report read, but were not actually returning it in the code